x-pack/plugins/observability_solution/slo/public/plugin.ts

```diff
@@ -94,6 +95,16 @@ export class SloPlugin
 
       const hasPlatinumLicense = license.hasAtLeast('platinum');
       if (hasPlatinumLicense) {
+        registerDashboardPanelPlacementSetting(
+          SLO_OVERVIEW_EMBEDDABLE_ID,
+          (serializedState: any) => {
```

Comment:
@nickpeihl I wanted to use `(serializedState: SloOverviewEmbeddableState)`, but typescript was complaining, because [here](https://github.com/elastic/kibana/blob/main/src/plugins/dashboard/public/dashboard_container/external_api/dashboard_panel_placement_registry.ts#L16) it expects an object. How shall I specify `SloOverviewEmbeddableState` (and not any) so that typescript doesn't complain?

Reply:
```suggestion
          (serializedState: SloOverviewEmbeddableState | undefined) => {
```

I think this would fix the type.

Reply:
And then maybe a conditional below that to return `{}` if serializedState is undefined.

```
if (!serializedState) return {};
```